RELEASE-NOTES.md

```diff
@@ -1,5 +1,11 @@
 # Wikibase DataModel release notes
 
+## Version 7.1.0 (2017-08-30)
+
+* `EntityIdValue::getArrayValue` can handle foreign entity IDs and entity IDs that do not have a numeric representation
```

Comment:
This is really minor and you can be changed whenever: this sentence does not follow the usual format/grammar of release notes. It does not explicitly say what changed

Reply:
true. rephrased. not sure it explains what changed well enough though.